ops/services/app_functions/report_stream_batched_publisher/functions/proxies.json

```diff
@@ -0,0 +1,4 @@
+{
+  "$schema": "http://json.schemastore.org/proxies",
+  "proxies": {}
+}
```

Comment:
Is this file necessary ?

Reply:
It was generated by the Azure Function App VSCode plugin when I created the project & function - I'd rather leave it than delete it and get burned